content/articles/getting-experiential-on-a-quarkus-project/index.md

```diff
@@ -0,0 +1,628 @@
+- **Imperative and Reactive development**: That is, it supports both synchronous and asynchronous operations.
+- **Standards and Libraries**: More Libraries built on known standards easily support full-stack development.
+
+### Comparison between Quarkus and other Java frameworks
+
+Java is a well-built and supported language. It has many available frameworks for large and small projects. Some include:
+
+- Spring
+- Spring Boot
+- Hibernate
+- Struts
+- Google Web Toolkit (GWT)
+- Blade
+- Grails
+- Quarkus
+- Micronaut
+- Microprofile among many more...
+
+Applications are build basing the choices of algorithms, data structures, and frameworks based on the performance and efficiency per requirements.
+
+You shall look at the similarities and differences between Quarkus and other Java frameworks such as Spring Boot, Micronaut, and Microprofile.
+
+When you compare Quarkus and Spring Boot, you will learn the following:
+The similarity is that Quarkus and Spring Boot both use Spring Web dependencies, have similar REST points, and are built using Maven and Gradle into JAR files.
+The differences are that Quarkus was built for performance and efficiency, in that it should take the least amount of memory and Boot and response time.
+Quarkus is a modern framework while Spring Boot has existed for a long time. This has made Quarkus to be optimized ,hence better performance.
+
+As for Quarkus and Micronaut, they are both modern frameworks build for microservices and serverless applications build for JDK and GraalVM.
+The key differences are that Quarkus relies more on Java Enterprises and Eclipse MicroProfile APIs, while Micronaut defines its APIs and is more based on the Spring and Grails frameworks.
+Quarkus also is based on an extension architecture which makes it integrate easily with third-party frameworks, while Micronaut has an Aspect-Oriented Programming (AOP) implementation that doesn't use reflection to define cross-cutting concerns.
+
+> **[GraalVM](https://www.graalvm.org/)**: a high-performance JDK distribution.
+> Is a high-performance Java Development Kit (JDK) distribution built to quicken the application executions written in Java and other JVM languages.
+> It provides other runtimes for Python, JavaScript, Ruby, and some other popular languages.
+> Availability of polyglot capabilities makes it possible to mix multiple programming languages in a single application while removing call costs of any foreign language.
+
+Lastly, as for Quarkus and Microprofile relationship, Microprofile is used to do microservices specification in the Java Enterprise environment.
+Quarkus on the other hand is an implementation of Microprofile which is well-built based on the given specifications.
+This makes them both compatible with each other.
+If you have worked with Microprofile, Quarkus is going to be simpler for you to work with.
```

Comment:
would you please consider doing the comparison in tabular or list form to bring out more clarity?

Reply:
i've decided to use the list form :+1:

Reply:
Done!